core/src/main/scala/org/apache/spark/sql/crossdata/catalog/CatalogChain.scala

```diff
@@ -44,20 +47,60 @@ object CatalogChain {
 private[crossdata] class CatalogChain private(val temporaryCatalogs: Seq[XDTemporaryCatalog],
```

Comment:
From this PR ownwards, the comment above should change. Yes, it is write through but not NWA, in fact it becomes WA